var/spack/repos/builtin/packages/gnutls/package.py

```diff
@@ -38,6 +38,7 @@ class Gnutls(AutotoolsPackage):
 
     variant("zlib", default=True, description="Enable zlib compression support")
     variant("guile", default=False, description="Enable Guile bindings")
+    variant("brotli", default=True, description="Enable brotli compression support")
```

Comment:
```suggestion
    variant("brotli", default=True, description="Enable brotli compression support", when="@3.7.4:")
```
https://gitlab.com/gnutls/gnutls/-/commit/5943dd3cd4e9b279156195d73af7ee068e709356